src/common/cache.py

```diff
@@ -90,3 +90,8 @@ def set_messages(self, entry_name, messages):
         with open(export_file_path, "w") as f:
             for msg in messages:
                 f.write(f"{json.dumps(msg.to_dict(serialize_datetimes_to_str=True))}\n")
+
+    def delete_file(self, entry_name):
+        file_path = f"{self.cache_dir}/{entry_name}.txt"
+        IOUtils.ensure_dirs_exist_for_file(file_path)
+        remove(file_path)
```

Comment:
Not all cache files are text files, so I think this should be def delete_file(self, filename)

Reply:
updated